examples/ExpoMessaging/package.json

```diff
@@ -1,42 +1,46 @@
 {
-  "main": "node_modules/expo/AppEntry.js",
   "scripts": {
-    "start": "expo start",
-    "android": "expo start --android",
-    "ios": "expo start --ios",
+    "start": "expo start --dev-client",
+    "android": "expo run:android",
+    "ios": "expo run:ios",
     "web": "expo start --web",
     "eject": "expo eject"
   },
   "dependencies": {
     "@react-native-community/masked-view": "0.1.10",
-    "@react-native-community/netinfo": "6.0.0",
+    "@react-native-community/netinfo": "7.1.3",
     "@react-navigation/native": "5.8.10",
     "@react-navigation/stack": "5.12.8",
-    "expo": "^42.0.0",
-    "expo-blur": "~9.0.3",
-    "expo-document-picker": "~9.2.4",
-    "expo-file-system": "~11.1.3",
-    "expo-haptics": "~10.1.0",
-    "expo-image-manipulator": "~9.2.2",
-    "expo-image-picker": "~10.2.2",
-    "expo-media-library": "~12.1.2",
-    "expo-sharing": "~9.2.1",
-    "react": "16.13.1",
-    "react-dom": "16.13.1",
-    "react-native": "https://github.com/expo/react-native/archive/sdk-42.0.0.tar.gz",
-    "react-native-gesture-handler": "~1.10.2",
-    "react-native-reanimated": "~2.2.0",
-    "react-native-safe-area-context": "3.2.0",
-    "react-native-screens": "~3.4.0",
+    "expo": "^44.0.0",
+    "expo-blur": "~11.0.0",
+    "expo-document-picker": "~10.1.3",
+    "expo-file-system": "~13.1.4",
+    "expo-haptics": "~11.1.0",
+    "expo-image-manipulator": "~10.2.0",
+    "expo-image-picker": "~12.0.1",
+    "expo-media-library": "~14.0.0",
+    "expo-sharing": "~10.1.0",
+    "react": "17.0.1",
+    "react-dom": "17.0.1",
+    "react-native": "0.64.3",
+    "react-native-gesture-handler": "~2.1.0",
```

Comment:
We also need to add GestureRootView, no? https://blog.expo.dev/expo-sdk-44-4c4b8306584a#:~:text=react%2Dnative%2Dgesture%2Dhandler%20now%20requires%20a%20GestureHandlerRootView%20wrapper.

Reply:
no, we don't need to. As our App.js is already using a`GestureHandlerRootView`